### PR TITLE
document how to post a post in languages other than English

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: authortechnical.Rmd [hunk 1]
```diff
@@ -402,6 +402,16 @@ Compare [raw markdown](https://raw.githubusercontent.com/ropensci/roweb3/HEAD/co
...
 
+## Multilingual posts
+
+**Please discuss your post's language with blog editors!**
+
+- Our website's default language is English so only posts in English are called `index.md`.
+- For **translating** a post, create a file in the same folder as the original post but called `index.<two-letter-language-code>.md`, for instance  `index.es.md`. Translate tags, title, description too.
+- For a post in say Spanish only, use the same folder structure as for an English post but call the post file `index.es.md`.
+- Add an author file in `content/author/author-name/_index.<two-letter-language-code>.md` even if it only duplicates `content/author/author-name/_index.md`.
+- For a post in a language without any post yet, please ask blog editors to notify the website team so we can add translation for the blog infrastructure (such as "Share this page" and similar phrases).
```
Comment (yabellini) [inline authortechnical.Rmd:413]:
```suggestion
- If you are writing the first blog entry for a language (yay!), please ask blog editors to notify the website team so we can add translation for the blog infrastructure (such as "Share this page" and similar phrases).
```

**Comment by steffilazerte** [comment]
Apologies for the delay (on this and the others!). I've made some suggestions, and pushed a commit myself to add more details to the Style guide as well as re-formatting a bit. Let me know what you think!

**Comment by steffilazerte** [comment]
Also, added the change you had in #215 while I was at it, so if you like these changes, we can delete that PR

**Review by steffilazerte — APPROVED** [review]
Looks good! Merge away.

**Pull request merged**